hack/build-cross.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+GOFLAGS=$1
+outputDir=$2
+
+rm -rf "${outputDir}"
```

Comment:
This can end up in error when the output directory does not exist, you should consider using `|| true` for both `rm` and `mkdir`.